pkg/kubectl/cmd/auth/cani.go

```diff
@@ -94,6 +95,9 @@ var (
 
 		# List all allowed actions in namespace "foo"
 		kubectl auth can-i --list --namespace=foo`)
+
+	resourceVerbs       = sets.NewString("get", "list", "watch", "create", "update", "patch", "delete", "deletecollection", "use", "bind", "impersonate", "*")
+	nonResourceURLVerbs = sets.NewString("get", "put", "post", "head", "options", "delete", "patch")
```

Comment:
include `*` in this list as well, in case someone was going to check if they had superuser permissions on non-resource URLs